__Testbase__/unit.test.js

```diff
@@ -0,0 +1,44 @@
+import Window from '../custom_modules/application/window';
+
+describe('Food Application ', () => {
+    test('Home page item count', () => {
```

Comment:
The test follows the AAA pattern, one thing though: consider below:

![image](https://user-images.githubusercontent.com/22321030/157267331-778f6faa-4364-49aa-8c52-cfcceae6d2e8.png)
 The `describe` could be "Food Items" and `test` could be "count items" to better describe the test. Dont you think?